playbook/run_consumer.go

```diff
@@ -19,6 +20,11 @@ func StartPlaybookRunConsumer(ctx api.Context) {
 }
 
 func EventConsumer(ctx api.Context) (int, error) {
+	tracer := otel.GetTracerProvider().Tracer("event-tracer")
```

Comment:
I think this will produce too many background traces - Maybe we can sample this at @ 1% ?

Reply:
Done